pkg/cmd/template/data_values_flags.go

```diff
@@ -274,26 +276,36 @@ func (s *DataValuesFlags) kvFile(kv string) (*datavalues.Envelope, error) {
 	return datavalues.NewEnvelopeWithLibRef(overlay, libRef)
 }
 
+// libraryRefAndKey separates a library reference and a key and validates that no libraryKeySep exist in the key.
+// libraryKeySep is disallowed in data value flag keys.
 func (DataValuesFlags) libraryRefAndKey(key string) (string, string, error) {
```

Comment:
```suggestion
func (DataValuesFlags) libraryRefAndKey(arg string) (string, string, error) {
```
Input is the whole argument, so `arg`?
2nd output is the key 👍🏻 

(not part of your changeset, but inspired by the comment below)